Ecosystem Partners/Marketplace Guides/gettting-started-with-biztalk-distributed.md

```diff
@@ -0,0 +1,229 @@
+
+{{{
+  "title": "Deploy Microsoft BizTalk Server 2013 R2 Server on Windows",
+  "date": "10/27/2015",
+  "author": "Terawe",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+
+<div class="row">
+  <a><img src="../../images/microsoft.png" width=300></a>
+</div>
+
+<div class="row">
+  <a><img src="../../images/terawe.png" width=250></a>
+</div>
+
+
+### Technology Profile
+
+BizTalk Server   is a Microsoft Server product for integration and connectivity with a multitude of systems, applications and services. BizTalk Server 2013 is the eight major  release of this enterprise ready product that can aid organizations tremendously in connecting disparate systems inside and outside the organization. It provides numerous adapters and has a robust, durable, and scalable messaging infrastructure. In addition to integration capabilities, BizTalk also provides strong durable messaging, a rules engine, EDI connectivity, Business Activity Monitoring (BAM), Windows Azure connectivity and RFID capabilities.
+
+### Description
+
+This KB article describes the options for installing and configuring BizTalk 2013 R2 Server in a distributed topology on the CenturyLink Cloud using blueprints.
+
+
+The following software is installed by the BizTalk Server 2013 R2 Distributed blueprint -
+
+- Microsoft SQL Server 2012 Standard
+
+- Microsoft BizTalk Server 2013 R2 Standard
+
+- Microsoft Enterprise Single Sign on
+
+The BizTalk Server 2013 R2 Distributed blueprint will add the required Windows Server Roles to enable a server to host an Active Directory (AD) and Domain Name Server (DNS).
+
+This blueprint requires an existing server running Microsoft Windows 2012 R2 Datacenter Edition 64-bit
+
+#### Blueprint - BizTalk Server 2013 R2 Distributed
+
+This blueprint installs SQL Server followed by Enterprise Single Sign on and finally the BizTalk Server 2013 R2.  Once the installs have completed, the BizTalk server is configured and a copy of the configuration command script is copied to the C:\Temp\ConfigureBTS.cmd for reference.
+
+### Audience
+
+CenturyLink Cloud Users
+
+### Impact
+
+After reading this article, the user should be able to deploy a BizTalk Server 2013 R2 Distributed blueprint on three existing servers running Microsoft Windows 2012 R2 Datacenter Edition 64-bit.
+
+### Prerequisite
+
+Access to the CenturyLink Cloud platform as an authorized user.
+
+#### Steps to Deploy Blueprint
+1. **Provision three new servers using the steps below**
+	1. Click 'CREATE SERVER' from the CenturyLink Control Portal Dashboard.  Refer to the image below for a sample.
+		<div class="row">
+		  <a><img src="../../images/BTSS1A.png" width=300></a>

```

Comment:
why is there HTML image links when this is mostly otherwise in markdown?  and your header has contentIsHTML": false.  image links should be in markdown across the board and sized appropriately directly